app/build.gradle

```diff
@@ -54,58 +51,62 @@ android {
     }
 
     sourceSets {
+        main { assets.srcDirs = ['src/main/assets'] }
+
         if (enable_plugin_kotlin) {
             main.java.srcDirs += 'src/main/kotlin'
         }
     }
 
-    compileOptions {
-        sourceCompatibility JavaVersion.VERSION_1_8
-        targetCompatibility JavaVersion.VERSION_1_8
-    }
-
+    flavorDimensions "default"
     productFlavors {
         flavorDefault {
         }
         flavorGplay {
             buildConfigField "boolean", "IS_GPLAY_BUILD", "true"
         }
         flavorTest {
-            applicationId "net.gsantner.memetastic_test"
+            applicationIdSuffix ".test"
```

Comment:
`_test` also possible? I know people running self-built nightly builds, which will make stuff un-updateable this way.

Reply:
Logically it shouldn't be a problem to just uninstall the old nightly and install the new nightly, but sure, I can make it ```_test``` instead.

Reply:
Actually it wouldn't matter, because the flavorTest applicationId is changing from "net.gsantner.memetastic" to "io.github.gsantner.memetastic". All "applicationIdSuffix" does is append a suffix onto the app's applicationId.